### ISSUE TITLE
Decrypt9 unable to build CIA from game installed in EmuNAND

### ISSUE THREAD

**Issue by KarayaOne** [d0k3/Decrypt9WIP#185]
Hey. 

I'm having a little problem with building the CIA files of two games, to be precise, they are Super Smash Bros and Pokemon Moon. 
Some details. 

System: o3DS
Entrypoint: Menuhax
CFW: Luma3DS
Decrypt9: Latest build. 

Now here is the thing, I setup my 3DS some time ago without the A9LH entry point (mainly because it was in its infancy at that time), even as A9LH got safer, I never really saw the reason to upgrade as my setup worked just fine. 
Now for some time my CFW was acting weird, and there is this new entry point; B9LH, which apparently, makes thing a lot easier. 
I decided that I would like to setup this new entry point. 

The problem was that I had several games on EmuNAND, downloaded from FreeShop. Having a limited bandwidth, I can afford to download them all again. I did not need the entire EmuNAND, so I decided I'd just dump the games using Decrypt9.

Now, for most games Decrypt9 worked fine. It built CIAs of most of games (using CIA Builder [EmuNAND/dec] method) just fine. The problem arises when I tried creating CIAs for Pokemon Moon and Smash Bros, and it gave my error. I tried multiple times with both orig & dec method, but it didn't work. 
I updated seeddb.bin, still didn't work. 

The error it gives me is something like this; 
Error decrypting file.. 
The file is too small, or the SD card is corrupted. 

This is quite strange as I dumped other games easily.

**Comment by KarayaOne** [comment]
Also I tried using GodMode9 (the 3dsx version, can't use the new one as don't have B9L/A9LH). The entrypoint I have is Menuhax so tried booting GodMode9 from Homebrew Launcher, however it always crashes and restarts HBL. Therefore, Decrypt9 is my only option for now.

**Comment by d0k3** [comment]
Alright... I may have a solution in the works that can help you running GodMode9 from your current setup, but that is not yet finished, so you'd have to wait.

For now, you could just go on with the boot9strap installation. Don't worry, your EmuNAND and all your installed games and savegames will be untouched by this (I'm the SafeB9SInstaller dev, I should know ;)). You can also just make a backup now (SysNAND/EmuNAND) to revert to your current state anytime.

**Comment by KarayaOne** [comment]
Hey. Thanks for the quick response. 

I have some GBA VC's (can't uninstall them because my luma got corrupted somehow, it won't let me boot into FBI that I installed over H&S app. So only solution is formatting the system, but then I'd have to setup MenuHax again which is a pain) installed on my SysNAND. Would they affect the installation of B9L or not? 
Basically. I want to install B9L on a new SD. 

Also I don't mind losing EmuNAND, it effectively got corrupted (won't let me open anything, gets stuck whenever I open anything). I backed up my saves so that's not an issue either. 
Also thanks. Waiting for that GodMode9 release.

**Comment by d0k3** [comment]
You can now run GodMode9 via OldLoader (look around in my repo). I'd say, though, uninstall menuhax and upgrade to B9S. B9S is installed to the NAND, not the SD card, and it will not touch anything of relevance to you.

Also, tell me if building CIAs for these two work from GodMode9!

**Comment by KarayaOne** [comment]
Ah. Thanks. 
I definitely plan on installing B9L, though I want to ask a quick question. 
The thing is that I have GBA VCs installed on sysNAND, if I install B9L, it won't cause any problem, right? Or do I uninstall GBA VCs first and then install B9L?

_Sent from my Motorola XT1254 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by d0k3** [comment]
No, definitely not. Leave everything as is, install b9s. Also, give me feedback on GM9 working with these twi titles.

**Comment by KarayaOne** [comment]
Thanks. 
I most definitely will, thing is that I blew the Power supply of my PC. Need to get a new one. Will update you tomorrow. 

_Sent from my Motorola XT1254 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by KarayaOne** [comment]
GodMode9 works with Old loader. Currently I'm trying to dump the two games. 

_Sent from my Motorola XT1254 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
